### PR TITLE
Fix(transifex): 🤖 Use conventional commit messages for transifex updates

### LINKED ISSUE #479: Use conventional commits for translation pushes
I am experimenting with conventional commits and automated releases and stumbled over the `[tx-robot] ...` commit messages.

To comply with https://www.conventionalcommits.org/en/v1.0.0/#summary it would be cool if the bot could send commits in the conventional format, e.g. `fix: Update translations from Transifex`.

Automation could classify that commit, generate a changelog and do the release.

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #479

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]

File: translations-app/handleAppTranslations.sh [hunk 1]
```diff
@@ -49,7 +49,7 @@ do
   tx migrate
   git add --force .tx/config
   rm .tx/config_*
-  git commit -am "[tx-robot] Update transifex configuration" -s || true
+  git commit -am "Fix(l10n): 🛠️ Update Transifex configuration" -s || true
```
Comment (ChristophWurst) [inline translations-app/handleAppTranslations.sh:52]: not sure about `Fix` vs `fix`

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

File: translations-app/handleAppTranslations.sh (hunk 1 above)
Comment (ChristophWurst) [reply translations-app/handleAppTranslations.sh:52]: https://www.conventionalcommits.org/en/v1.0.0/#are-the-types-in-the-commit-title-uppercase-or-lowercase

Comment (nickvergessen) [reply translations-app/handleAppTranslations.sh:52]: he units of information that make up Conventional Commits MUST NOT be treated as case sensitive by implementors, with the exception of BREAKING CHANGE which MUST be uppercase.

**Comment by skjnldsv** [comment]
Really not convinced about the emojis @nickvergessen  :/
Especially in commits titles, it doesn't adds anything bu noise imho, especially with terminals that are not all able to render those  :see_no_evil:

**Pull request merged**